contribute/en/Release-Process.md

```diff
@@ -345,10 +345,10 @@ Please find all the details in the [Changelog](https://github.com/jamulussoftwar
 _Windows users: Please note that in the first days after release SmartScreen will probably display warnings about this release being unknown upon download and/or execution of the installer. Let us know when you do not see this warning anymore and we will update this announcement accordingly._
 
 **[↓ Windows](<!-- direct link to Windows version -->)** (ASIO version), alternative: [↓ JACK version](<!-- direct link to JACK version -->)
-**[↓ macOS (Intel)](<!-- direct link to macOS SIGNED Intel version -->)** for Catalina (10.15) and higher and [↓ macOS legacy build](<!-- direct link to macOS legacy version -->) (unsigned) for macOS Mojave (10.14) down to Yosemite (10.10).
-**[↓ macOS (Apple Silicon)](<!-- direct link to macOS SIGNED ARM version -->)** for Big Sur (11.0) or later (needs ARM chip)
+**[↓ macOS (Universal)](<!-- direct link to macOS SIGNED Intel version -->)** for Catalina (10.15) and higher and [↓ macOS legacy build](<!-- direct link to macOS legacy version -->) (unsigned) for macOS Mojave (10.14) down to Yosemite (10.10).
```

Comment:
Change the comment... the link is _not_ to the signed _Intel_ version if it's the Universal build.

Reply:
I think starting from Big Sur, the OS supports Apple silicon. Before that, the universal build just runs like the previous Intel version.

Reply:
Right, that makes sense.  It might be worth something like
```suggestion
**[↓ macOS (Universal)](<!-- direct link to macOS SIGNED universal version -->)** for Catalina (10.15, Intel only) or Big Sur (11.0) and higher and [↓ macOS legacy build](<!-- direct link to macOS legacy version -->) (unsigned) for macOS Mojave (10.14) down to Yosemite (10.10).
```

Reply:
There aren't any < Big Sur ARM Macs released to the public, so I don't think we need to specify it.

Reply:
Fixed the comment.